examples/inference/analytic-normal2d/normal2d.ini

```diff
@@ -2,9 +2,7 @@
 name = test_normal
 
 [sampler]
-name = emcee
-nwalkers = 5000
-niterations = 100
+name = ultranest
```

Comment:
I think you accidentally committed this. Please don't change this example script; better to add a new one if you want to show one using ultranest.